sdk/cosmosdb/cosmos/test/public/integration/crossPartition.spec.ts

```diff
@@ -183,8 +183,12 @@ describe("Cross Partition", function (this: Suite) {
         Math.abs(fetchAllResponse.requestCharge - totalExecuteNextRequestCharge) /
         totalExecuteNextRequestCharge;
       assert(
-        percentDifference <= 0.01,
-        "difference between fetchAll request charge and executeNext request charge should be less than 1%"
+        percentDifference <= 0.1,
```

Comment:
@v1k1 This PR discusses the management of RU charge format in the feed response. However, we are currently modifying the percentage difference in RUs. Could you assist me in comprehending this?